kimai2-cmd.js

```diff
@@ -554,6 +565,18 @@ function uiSelectMeasurement(thelist) {
     })
 }
 
```

Comment:
Please add JSDoc to all new functions. You don't have to overthink it, just some words like other functions. It will make reading the code much easier. https://en.wikipedia.org/wiki/JSDoc

Reply:
Done, thanks for your feedback!